rpc/v8/subscriptions_test.go

```diff
@@ -882,107 +882,6 @@ func TestSubscribePendingTxs(t *testing.T) {
 	})
 }
 
-func TestUnsubscribe(t *testing.T) {
```

Comment:
These tests actually server the `starknet_unsubscribe`  instead of `juno_unsubscribe`. I think we'll eventually deprecate `juno_unsubscribe` because it's not currently in v8, so we shouldn't move this test.